tests/UI/pages/BO/orders/add.js

```diff
@@ -246,11 +335,33 @@ class AddOrder extends BOBasePage {
   /**
    * Set order status
    * @param page {Page} Browser tab
-   * @param orderStatus {{id: number, status: string}} Order status to choose
+   * @param orderStatus {string} Order status to choose
    * @returns {Promise<void>}
    */
   async setOrderStatus(page, orderStatus) {
-    await this.selectByVisibleText(page, this.orderStatusSelect, orderStatus.status);
+    await this.selectByVisibleText(page, this.orderStatusSelect, orderStatus);
```

Comment:
I think this could break some old tests (ex: `UI/campaigns/functional/BO/02_orders/01_orders/04_checkInvoiceDownloadedFromList.js`)

Reply:
The test `checkInvoiceDownloadedFromList` use `setOrderStatus` method from `ordersPage` and not `addOrderPage`.

Reply:
I check the other tests

Reply:
My mistake sorry